modules/ROOT/pages/actor-filter-archetype.adoc

```diff
@@ -189,9 +192,14 @@ The implementation sources contain all the logic of the actor filter:
 * The validation of the inputs
 * The execution of the business logic to filter the users for a given actor.
 
-The archetype offers the possibility to generate the default sources in Java, Groovy or Kotlin. The build result will always be a Java archive (jar), no matters the langage selected.
+The archetype offers the possibility to generate the default sources in Java, Groovy or Kotlin. The build result will always be a Java archive (jar) and some Zip archives (assemblies), no matters the language selected.
+
+The entry point of the implementation sources must extend the https://javadoc.bonitasoft.com/api/{varVersion}/org/bonitasoft/engine/filter/AbstractUserFilter.html[__AbstractUserFilter__, window="_blank"] class
+or implement https://javadoc.bonitasoft.com/api/{varVersion}/org/bonitasoft/engine/filter/UserFilter.html[__UserFilter__, window="_blank"] interface.
+
+===== Add multiple implementations
 
-The entry point of the implementation sources must extend the class _`org.bonitasoft.engine.filter.AbstractUserFilter`_.
+To add a new implementation create a `.impl` file in the `src/main/resources-filtered/` folder. They will be added to the `all` zip archive at build time. It is recommended to use the Maven properties for implementation id and version.
```

Comment:
The 'all' zip archive isn't created anymore starting the 2021.2 version, we should update this paragraph for this version later.

Reply:
Yes